src/Spryker/Zed/EventBehavior/Persistence/Propel/Behavior/EventBehavior.php

```diff
@@ -36,7 +36,7 @@ public function preSave()
     public function postSave()
     {
         return "
-\$this->addSaveEventToMemory();
+\$this->logSavedEventBehaviorEntityChange();
```

Comment:
it's not logging, it's an event entity which will be used for triggering, maybe this one `saveEventBehaviorEntityChange`

Reply:
There's already `saveEventBehaviorEntityChange` method, but I can change it's name, but what about the post delete method?

Maybe  `saveAddedEventBehaviorEntityChange` and `saveDeleteedEventBehaviorEntityChange`?